src/controller/content/content.controller.ts

```diff
@@ -117,6 +117,7 @@ export class ContentController {
     description: 'type id',
   })
```

Comment:
Please confirm with the frontend team. These APIs are used exclusively on the ADMIN side. I've checked most of the APIs from the ADMIN side,